cmd/fpt/run.go

```diff
@@ -22,7 +22,7 @@ import (
 //   3. Print log as we go (if --tail option)
 //   4. Print escalation status as we go (if --tail option)
 //   5. Cleanup (stop applied policy, delete policy template)
-func policyTemplateRun(ctx context.Context, cli policy.Client, file string, runOptions []string, keep bool, dryRun bool, noLog bool) error {
+func policyTemplateRun(ctx context.Context, cli policy.Client, file string, runOptions []string, runCredentials string, keep bool, dryRun bool, noLog bool) error {
```

Comment:
I think it would be easier to use if we can provide credentials similar to options. If there are multiple creds currently we have to do key1-value1,key2=value. This is as simple as setting run credentials to []string.

Reply:
The problem I ran into was how to differentiate between parameter arguments and credential arguments on the command line. The solution I came up with was to allow the user to supply credentials as a flag, taking the form `--credentials=<name1>=<val1>,<name2>=<val2>`. I wasn't sure there was a use-case for supplying more than one credential pair, so I simplified the code by removing the option for multiple credentials. I can add that option back in if you think it's good to have.

Reply:
We do support passing multiple credentials. We can pass creds like `--credentials cred1=value1 cred2=value2`.